src/support/timestamp.c

```diff
@@ -198,7 +198,7 @@ __time_aggregate_validate_parent(
           __wt_time_aggregate_to_string(ta, time_string[0]),
           __wt_time_aggregate_to_string(parent, time_string[1]));
 
-    if (ta->oldest_start_ts < parent->oldest_start_ts)
+    if (ta->oldest_start_ts != WT_TS_NONE && ta->oldest_start_ts < parent->oldest_start_ts)
```

Comment:
I am surprised with this fix, we must be hitting this code path a lot in the current develop branch, i.e every time `oldest_start_ts` is `0`, which results in false positives. Am I correct?

Reply:
No. When the child `oldest_start_ts` is 0, then the parent `oldest_start_ts` will also be 0. We assert only when the child is having a smaller value than the parent.